.storybook/preview.tsx

```diff
@@ -14,19 +14,41 @@
  *    limitations under the License.
  */
 
-import { BaseStyling } from "../src/_BaseStyling_/BaseStyling";
+import React, { useEffect } from "react";
 import "./base.scss";
+import { BaseStyling } from "../src/_BaseStyling_/BaseStyling";
+import { useState } from "react";
 
 // https://storybook.js.org/docs/react/writing-stories/parameters#global-parameters
 export const parameters = {
   // https://storybook.js.org/docs/react/essentials/actions#automatically-matching-args
   actions: { argTypesRegex: "^on.*" }
 };
 
-export const decorators = [
-  Story => (
-    <BaseStyling>
-      <Story />
-    </BaseStyling>
-  )
-];
+const Preview = story => {
+  const [properties, setProperties] = useState({});
+
+  useEffect(() => {
+    parseBaseStylingStorage();
+  }, []);
+
+  useEffect(() => {
+    window.addEventListener("updated-styling", parseBaseStylingStorage);
+
+    return () => {
+      window.removeEventListener("updated-styling", parseBaseStylingStorage);
+    };
+  }, []);
+
+  const parseBaseStylingStorage = () => {
+    const baseStylingSessionStorage = window.sessionStorage.getItem("basestyling");
+
+    if (baseStylingSessionStorage !== "undefined" && baseStylingSessionStorage) {
```

Comment:
```suggestion
    if (baseStylingSessionStorage && baseStylingSessionStorage !== "undefined") {
```
seams more readable, don't u think so?